src/new-client/src/App/Trades/TradesState/filterState.ts

```diff
@@ -47,6 +89,11 @@ interface FilterEvent {
 interface ColFieldToggle extends FilterEvent {
   value: unknown
 }
+
+interface NumFieldToggle extends FilterEvent {
```

Comment:
I think is less a toggle than a "NumFilterSet".  Toggling "off" is the reset event, right?